solutions/usaco-945.mdx

```diff
@@ -0,0 +1,73 @@
+---
+id: usaco-945
+source: USACO Gold 2019 US Open
+title: Snakes
+author: Sofia Yang
+---
+
+[Official Analysis and Code(C++)](http://www.usaco.org/current/data/sol_snakes_gold_open19.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^3)$
+
+<LanguageSection>
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class snakes {
```

Comment:
```suggestion
public class Snakes
```

Reply:
did you mean to delete the opening brace?